### ISSUE TITLE
Terrain System

### ISSUE THREAD

**Issue by AMZN-Liv** [o3de/o3de#1847]
**Description**
As a content author using Open 3D Engine, I want to be able to create richly populated, dynamic, realistic, outdoor environments. Using O3DE's built-in authoring tools, I can create outdoor environments with expansive terrains, a variety of vegetation elements that interact with weather systems, various types of water elements (oceans, lakes, rivers), the ability to place and generate random assortments of prefabs, and authoring tools that allow me to place and manipulate elements in my environment. 

Once completed, this system will enable users to:

•	Import height, color, and surface data
•	Assign that data to regions of the virtual world
•	Render the data as a textured terrain mesh
•	Represent the terrain in PhysX as a physics surface
•	Query the terrain at runtime via an API to get height, color, and surface data at specific points or regions

**Status**
RFC posted: https://github.com/o3de/sig-content/issues/4

**Discussion**
https://github.com/o3de/o3de/discussions/1974

**Comment by codesnip** [comment]
+1

**Comment by NiklasHenricson** [comment]
Is this being followed up still? Can't wait! :)

**Comment by AMZN-Liv** [comment]
Yes! The first iteration of the terrain gem is in the development branch and will be available in main with the branch we are currently stabilizing (https://github.com/o3de/o3de/tree/stabilization/2110)
